core/src/main/java/org/elasticsearch/ExceptionsHelper.java

```diff
@@ -78,7 +78,7 @@ public static Throwable unwrapCause(Throwable t) {
             }
             if (counter++ > 10) {
                 // dear god, if we got more than 10 levels down, WTF? just bail
-                logger.warn("Exception cause unwrapping ran for 10 levels...", t);
+                logger.warn("Exception cause unwrapping ran for 10 levels: {}", t.getMessage());
```

Comment:
I think we'd do better logging the first 10 levels or something. It'd be fairly painful not to have any stack trace.